chatkit-core/src/main/kotlin/com/pusher/chatkit/state/ChatkitState.kt

```diff
@@ -1,6 +1,37 @@
 package com.pusher.chatkit.state
 
 import com.pusher.chatkit.rooms.state.JoinedRoomsState
+import com.pusher.chatkit.rooms.state.joinedRoomReducer
+import com.pusher.chatkit.rooms.state.joinedRoomsReceivedReducer
+import com.pusher.chatkit.rooms.state.leftRoomReducer
+import com.pusher.chatkit.rooms.state.roomDeletedReducer
+import com.pusher.chatkit.rooms.state.roomUpdatedReducer
+import org.koin.core.module.Module
+import org.koin.core.qualifier.named
+import org.koin.dsl.module
+import org.reduxkotlin.combineReducers
+import org.reduxkotlin.createStore
+
+internal val createStoreModule: () -> Module = {
+    module {
+        single(named(store::subscribe.name)) { store.subscribe }
+        single { store.getState }
+
+        // TODO: uncomment when reducers branch is merged into v2 and this branch caught up
```

Comment:
the reducers branch has already been merged in to v2 :-)

Reply:
[Done](https://github.com/pusher/chatkit-android/pull/195/commits/613486755da1cbdc189c70f405ed3eeb1f504a54) [Done](https://github.com/pusher/chatkit-android/pull/195/commits/c244f265fb5c0c5784a89d98dd15bd60cf48ee42)